src/matrix.jl

```diff
@@ -45,7 +45,6 @@ Base.@propagate_inbounds function Base.setindex!(
     i::Integer,
     j::Integer,
 )
-    @boundscheck checkbounds(A, i, j)
```

Comment:
hmm, why removing those checkbounds? are we checking those in `ref` as well?

Reply:
Exactly, the `ref` method already checks bounds for both `ArbVector` and `ArbMatrix`.

Reply:
I only happened to notice them because I went through all occurrences of `setindex!`.